tests/integration_test/e2e_test.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group('FlutterFire', () {
+    test('dummy test', () {
+      expect(true, true);
+    });
+  });
```

Comment:
Should this PR be in Draft? I feel it's not ready to be merged 😅

Reply:
This is intended because I'm not merging into `master`. I'm merging into `migrate-to-integration-tests` because I'm splitting the PRs into multiple small PRs. Therefore, it's easier to review for you :) The PRs with the migration for every package will follow in the next weeks 👍

Reply:
Oh thanks didn't checked that, sorry 😅